src/main/kotlin/gg/essential/elementa/components/ScrollComponent.kt

```diff
@@ -95,31 +143,45 @@ class ScrollComponent @JvmOverloads constructor(
     val verticalOverhang: Float
         get() = max(0f, calculateActualHeight() - getHeight())
 
+
+
+    val mouseScrollLambda: UIComponent.(UIScrollEvent) -> Unit = {
+        if (Window.of(this).version >= ElementaVersion.v5) {
+            // new behavior
+            val scrollDirection = if (!UKeyboard.isShiftKeyDown()) primaryScrollDirection else secondaryScrollDirection
+            scrollDirection?.let { direction ->
+                if (!onScroll(it.delta.toFloat(), isHorizontal = direction == Direction.Horizontal) && passthroughScroll) {
+                    getNextHighestScrollComponent()?.fireScrollEvent(it)
```

Comment:
Do we need to manually forward this? Could we not just not call `stopPropagation` so the event naturally bubbles up (and also gives non-scroll components in between a chance to look at it)?